src/amo/experiments/20210531_download_funnel_experiment.js

```diff
@@ -0,0 +1,11 @@
+/* @flow */
+export const VARIANT_CURRENT = 'current-link';
+export const VARIANT_NEW = 'new-link';
+
+export const EXPERIMENT_CONFIG = {
+  id: '20210531_download_funnel_experiment',
+  variants: [
+    { id: VARIANT_CURRENT, percentage: 0.5 },
+    { id: VARIANT_NEW, percentage: 0.5 },
```

Comment:
I thought we had a discussion where we weren't going to show experiments on 50% of the users, more like 10%?  (If I'm misremembering, ignore this)

Reply:
We discussed that as a possible mitigation for the performance issue created by having content change on the client after receiving the original html from the server. For this experiment, there is no visual content difference - it's only the destination of the link that is different for each cohort, so I figured we might as well just go 50/50.